prometheus rules not taken into account

<!--
 Before you open the bug report please review the following troubleshooting guide:

 - [Troubleshoot Bitnami Helm Chart Issues](https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues)
 -->

**Which chart**:
bitnami/mongodb v9.3.1

**Describe the bug**
The prometheus rules added in the values.yaml file are not taken into account.

**To Reproduce**

1. In values.yaml file:
```
  prometheusRule:
    enabled: false
    additionalLabels:
      release: monitoring
    namespace: monitoring
    rules:
    - alert: MongodbReplicationLag
      expr: avg(mongodb_mongod_replset_member_optime_date{state="PRIMARY"}) - avg(mongodb_mongod_replset_member_optime_date{state="SECONDARY"}) > 10
      for: 5m
      labels:
        severity: critical
      annotations:
        summary: MongoDB replication lag
        description: Mongodb replication lag is more than 10s
```
2. helm upgrade mongodb bitnami/mongodb -f mongodb/values.yml
```
coalesce.go:196: warning: cannot overwrite table with non table for rules (map[])
coalesce.go:196: warning: cannot overwrite table with non table for rules (map[])
```
... normal helm deployment

**Expected behavior**
A PrometheusRules resource should be created with all the rules defined in it.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.3.4", GitCommit:"a61ce5633af99708171414353ed49547cf05013d", GitTreeState:"clean", GoVersion:"go1.14.9"}
```

- Output of `kubectl version`:

```Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2", GitCommit:"f5743093fd1c663cb0cbc89748f730662345d44d", GitTreeState:"clean", BuildDate:"2020-09-16T13:41:02Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"dff82dc0de47299ab66c83c626e08b245ab19037", GitTreeState:"clean", BuildDate:"2020-07-15T16:51:04Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
A possible reason could come from the fact that metrics.prometheusRule.rules is expecting a dictionary in the template instead of a list of dictionary.

--- a/bitnami/mongodb/Chart.yaml
+++ b/bitnami/mongodb/Chart.yaml
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.0.0
+version: 10.0.1

--- a/bitnami/mongodb/README.md
+++ b/bitnami/mongodb/README.md
@@ -510,6 +510,27 @@ As the image run as non-root by default, it is necessary to adjust the ownership
 
 As an alternative, this chart supports using an initContainer to change the ownership of the volume before mounting it in the final destination. You can enable this initContainer by setting `volumePermissions.enabled` to `true`.
 
+## Using Prometheus rules
+
+You can use custom Prometheus rules for Prometheus operator by using the `prometheusRule` parameter, see below a basic configuration example:
+
+```yaml
+metrics:
+  enabled: true
+  prometheusRule:
+    enabled: true
+    rules:
+    - name: rule1
+      rules:
+      - alert: HighRequestLatency
+        expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
+        for: 10m
+        labels:
+          severity: page
+        annotations:
+          summary: High request latency
+```
+
 ## Enabling SSL/TLS
 
 This container supports enabling SSL/TLS between nodes in the cluster, as well as between mongo clients and nodes, by setting the MONGODB_EXTRA_FLAGS and MONGODB_CLIENT_EXTRA_FLAGS environment variables, together with the correct MONGODB_ADVERTISED_HOSTNAME.

--- a/bitnami/mongodb/templates/prometheusrule.yaml
+++ b/bitnami/mongodb/templates/prometheusrule.yaml
@@ -10,5 +10,6 @@ metadata:
     {{- end }}
 spec:
   groups:
-    {{- include "common.tplvalues.render" (dict "value" .Values.metrics.prometheusRule.rules "context" $) | nindent 4 }}
+  - rules:
+      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.prometheusRule.rules "context" $) | nindent 4 }}
 {{- end }}

--- a/bitnami/mongodb/templates/prometheusrule.yaml
+++ b/bitnami/mongodb/templates/prometheusrule.yaml
@@ -10,6 +10,5 @@ metadata:
     {{- end }}
 spec:
   groups:
-  - rules:
-      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.prometheusRule.rules "context" $) | nindent 4 }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.metrics.prometheusRule.rules "context" $) | nindent 2 }}
 {{- end }}

--- a/bitnami/mongodb/values-production.yaml
+++ b/bitnami/mongodb/values-production.yaml
@@ -90,7 +90,7 @@ auth:
   # existingSecret: name-of-existing-secret
 
 tls:
- ## Enable or disable MongoDB TLS Support 
+ ## Enable or disable MongoDB TLS Support
   enabled: false
  ##
  ## Bitnami Nginx image
@@ -913,8 +913,20 @@ metrics:
     ## Specify the namespace where Prometheus Operator is running
     ##
     # namespace: monitoring
+
     ## Define individual alerting rules as required
     ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#rulegroup
     ##      https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/
+    ##
+    ## This is an example of a rule, you should add the below code block under the "rules" param, removing the brackets
+    ## - name: example
+    ##   rules:
+    ##   - alert: HighRequestLatency
+    ##     expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
+    ##     for: 10m
+    ##     labels:
+    ##       severity: page
+    ##     annotations:
+    ##       summary: High request latency
     ##
     rules: {}

--- a/bitnami/mongodb/values.yaml
+++ b/bitnami/mongodb/values.yaml
@@ -913,8 +913,20 @@ metrics:
     ## Specify the namespace where Prometheus Operator is running
     ##
     # namespace: monitoring
+
     ## Define individual alerting rules as required
     ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#rulegroup
     ##      https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/
+    ##
+    ## This is an example of a rule, you should add the below code block under the "rules" param, removing the brackets
+    ## - name: example
+    ##   rules:
+    ##   - alert: HighRequestLatency
+    ##     expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
+    ##     for: 10m
+    ##     labels:
+    ##       severity: page
+    ##     annotations:
+    ##       summary: High request latency
     ##
     rules: {}

